src/state/upload/upload-reducers.js

```diff
@@ -37,9 +38,24 @@ export default function (state = initialState, action) {
 
         case uploadActionTypes.FETCH_ITEMS_SUCCEEDED: {
             const { groups } = action.data;
+            let ID = null;
```

Comment:
it's basically to show the meta data for generic on page load without having to click the checkbox